spring-core/src/main/java/org/springframework/asm/ClassWriter.java

```diff
@@ -283,7 +283,7 @@ public final void visit(
     if (signature != null) {
       this.signatureIndex = symbolTable.addConstantUtf8(signature);
     }
-    this.superClass = superName == null ? 0 : symbolTable.addConstantClass(superName).index;
+    this.superClass = (superName == null ? 0 : symbolTable.addConstantClass(superName).index);
```

Comment:
Please refrain from modifying classes under `org.springframework.asm`, `org.springframework.cglib`, and `org.springframework.objenesis`. Those include repackaged forks of the third-party libraries ASM, CGLIB, and Objenesis. Any refactoring to those classes should take place upstream in the originating repository. The Spring Framework will then pick up the changes when syncing with official updates of the forked third-party libraries.

----

There is no need to make a change to this PR since I have already reverted the changes to ASM code locally.

Reply:
@sbrannen 
Thank you for your kind answer!!